Sources/Auth/AuthClient.swift

```diff
@@ -1,5 +1,28 @@
 import Foundation
 
 class AuthClient {
+    enum Errors: Error {
+        case malformedPairingURI
+    }
 
+    private let appPairService: AppPairService
+
+    private let walletPairService: WalletPairService
+
+    init(appPairService: AppPairService, walletPairService: WalletPairService) {
+        self.appPairService = appPairService
+        self.walletPairService = walletPairService
+    }
+
+    func connect() async throws -> String {
```

Comment:
if you look at specs, we do not have a `connect()` method. We have `request()` instead that is suppose to do as follows:
```
appPairService.create()
authRequestService().request
return uri
```

Reply:
Like this? 
```
    func request(params: RequestParams) async throws -> String {
        let uri = try await appPairService.create()
        try await appRequestService.request(params: params, topic: uri.topic)
        return uri.absoluteString
    }
```